src/discord/globals.py

```diff
@@ -2,19 +2,16 @@
 Holds global variables shared between cogs and variables that are initialized when
 the bot is first setup.
 """
-import os
 
-from dotenv import load_dotenv
+from env import env
 
-load_dotenv()
-
-TOKEN = os.getenv("DISCORD_TOKEN")
-DEV_TOKEN = os.getenv("DISCORD_DEV_TOKEN")
-dev_mode = os.getenv("DEV_MODE") == "TRUE"
+TOKEN = env.discord_token
+DEV_TOKEN = env.discord_dev_token
+dev_mode = env.dev_mode
```

Comment:
Since these variables don't change, maybe it would just make sense to replace all instances of `TOKEN` with `env.discord_token`, all instances of `DEV_TOKEN` with `env.discord_dev_token` and so on?

Reply:
Would these lines also be subject to the same change as well? We can set default values within the validator as well.
`BOT_PREFIX` is probably the only one here that I'd keep in `globals.py` here but I thought I'd just mention it.
https://github.com/cbrxyz/pi-bot/blob/deb8b7aead43222a1857bdc764ca15deb6296d02/src/discord/globals.py#L13-L15
https://github.com/cbrxyz/pi-bot/blob/deb8b7aead43222a1857bdc764ca15deb6296d02/src/discord/globals.py#L30-L31

Reply:
Yup, even `BOT_PREFIX` can probs just go in `bot.py`